<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddCompanyScreen</class>
 <widget class="QMainWindow" name="AddCompanyScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>786</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>786</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>786</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Roko d.o.o. ponude - version 1.0</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="head_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>1001</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Dodavanje nove tvrtke</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="name_label">
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>142</y>
      <width>101</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Naziv tvrtke:</string>
    </property>
   </widget>
   <widget class="QLabel" name="address_label">
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>261</y>
      <width>111</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Adresa tvrtke:</string>
    </property>
   </widget>
   <widget class="QLabel" name="oib_label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>381</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>OIB:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="name_edit">
    <property name="geometry">
     <rect>
      <x>186</x>
      <y>142</y>
      <width>541</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="address_edit">
    <property name="geometry">
     <rect>
      <x>198</x>
      <y>261</y>
      <width>531</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="oib_edit">
    <property name="geometry">
     <rect>
      <x>194</x>
      <y>381</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="add_button">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>520</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Dodaj</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back_button">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>520</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Natrag</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
